tests/kernel_bundle/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ set(independent_cases_list
   get_kernel_bundle_zero_kernels.cpp
   get_kernel_bundle_with_kernel_attr.cpp
   get_kernel_bundle_without_kernel_attr.cpp
+  kernel_bundle_api.cpp
```

Comment:
Just for my own education: why `kernel_bundle_api` is this list, but `kernel_bundle_spec_const` from #493 is not?

Reply:
Including kernel_bundle_api in the list is my mistake. Fixed it, thanks.